.eslintrc.js

```diff
@@ -27,12 +32,29 @@ module.exports = {
     '@operation_code/custom-rules',
     'import',
     'lodash',
+    '@typescript-eslint',
   ],
   globals: {
     cy: true,
     Cypress: true,
   },
   overrides: [
+    {
+      extends: ['plugin:@typescript-eslint/disable-type-checked'],
+      files: ['./**/*.js'],
+    },
+    {
+      files: ['./**/*.ts', './**/*.tsx'],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        project: ['./tsconfig.json'],
```

Comment:
i think you just need
`project: true`

Reply:
At first, I had this parserOptions in the overrides with project set to true or the tsconfig but it kept complaining that it couldn't find the parserServices when linting.